nixos/modules/services/web-apps/gancio.nix

```diff
@@ -54,24 +54,20 @@ in
           };
           baseurl = mkOption {
             type = types.str;
-            default = "";
-            example = "/gancio";
+            default = "http${
+              lib.optionalString config.services.nginx.virtualHosts."${cfg.settings.hostname}".enableACME "s"
+            }://${cfg.settings.hostname}";
+            defaultText = lib.literalExpression ''"https://''${cfg.settings.hostname}"'';
+            example = "https://demo.gancio.org/gancio";
             description = "The URL path under which the server is reachable.";
```

Comment:
Could you update the description as well? This is an entire URL, not a URL path.